README.md

```diff
@@ -1 +1,56 @@
-# Javascript-group-capstone-project
\ No newline at end of file
+#  JavaScript Capstone Project
+> In this project we built a movie api that displays movies to the user
+
+# Built With
+
+- Html
+- Css
+- Js
+- Git & Gitflow
+- webpack
+- TvMaze api 
+
+
+
+## Getting Started
+To run the above project you need to meet the following requirements:
+
+
+## Prerequisites
+- Have [Vscode](https://code.visualstudio.com/) installed 
+- Have [Git](https://git-scm.com/) installed
+- Node installed
+
+## Screenshot of our Project
+
+<img width="1280" alt="Screen Shot 2022-04-23 at 13 23 47" src="https://user-images.githubusercontent.com/91213045/164892432-9a161551-afe5-477f-8ac2-c3b74501388b.png">
+
+## Deployment
+
+[link to our project](https://yannickzahinda.github.io/Javascript-group-capstone-project/)
+
+## Run
+> To run the above project :
+> Run the following commands on your terminal of choice
+
+- git clone 
+- cd /dir
```

Comment:
- [ ] Please update the lines 36 and 37 to `git clone https://github.com/YannickZahinda/Javascript-group-capstone-project.git` and `cd Javascript-group-capstone-project` as mentioned by @ShoiraTa. Because this makes it more informative.